vendor/engines/secure_rooms/app/support/secure_rooms/auto_orphan_occupancy.rb

```diff
@@ -25,7 +25,7 @@ def long_running_occupancies
     def orphan_occupancy(occupancy)
       occupancy.mark_orphaned!
       SecureRooms::AccessHandlers::OrderHandler.process(occupancy)
-      MoveToProblemQueue.move!(occupancy.order_detail) if occupancy.order_detail
+      MoveToProblemQueue.move!(occupancy.order_detail, cause: :orphaned) if occupancy.order_detail
```

Comment:
I think this is probably fine, though I could see NU asking for to distinguish between these two orphan cases. I think this one looks like it's similar to the `auto_expire`, so we could all this one `auto_orphan` if we need to change it. The other one I feel like could have multiple context and figuring those out would be a pain.